### ISSUE TITLE
Note content not sanitized; code snippets in notes can break script

### ISSUE THREAD

**Issue by davidwolfpaw** [viko16/boost2fs#1]
While running b2f on a file that has some PHP commands and errors in a snippet, the following error comes up:

`ENOENT: no such file or directory, open (directory and file portion with snippet)`

My suggestion would be some method of sanitizing the input form the files when reading them from the disk.

**Comment by viko16** [comment]
Hi David,
Can you give some cases ?
I can not reproduce it. 😢

**Comment by davidwolfpaw** [comment]
@viko16 Sure, thanks for following up.

I have a BoostNote file that I had pasted a PHP error log item into. The entire file contents:
```
Warning: Use of undefined constant ‘APP_HTTPS’ - assumed '‘APP_HTTPS’' (this will throw an Error in a future version of PHP) in /home/orangebl/public_html/iwp/config.php on line 21
```
The below is the error getting thrown.
```
~/Dropbox/Apps/Boostnote
╰─$ b2f .
  Found 71 notes.
✖ ENOENT: no such file or directory, open '/Users/david/Dropbox/Apps/Boostnote/out/Personal Posts/Warning: Use of undefined constant ‘APP_HTTPS’ - assumed '‘APP_HTTPS’' (this will throw an Error in a future version of PHP) in /home/orangebl/public_html/iwp/config.php on line 21.md'
```
At that point the process ends and only files that were processed before reaching that one are completed. By removing that note and running again I was able to complete the conversion.

**Comment by viko16** [comment]
Thank you @davidwolfpaw so much!

I have fixed in v0.4.0

Side Effect: To sanitized filenames (notes title), it will remove directory paths and invalid characters. So files that have been converted, may be generate a new one. 😨
